config/locales/countries/mediterranean/en.yml

```diff
@@ -2,6 +2,13 @@ en:
   countries:
     mediterranean:
       title: Mediterranean
+      bounding_box:
```

Comment:
This doesn't feel like it should live in a locale file as it's the same across languages and not related to translations :) 
I would put it into a separate yml file related to countries data if we want to stick with ymls.

However, I would be tempted to say that this should be a database attribute for the GeoEntity model; so something to be imported with other data when seeding the database.

Reply:
I think you're right that it belongs as a db attribute. Looks like it would maybe need a new csv? Would this be something you could set up quickly? If not I can give it a go and get your review.

Reply:
I can quickly set it up on this PR :) no problem!

Reply:
Thank you!